marbot-aws-account-connection.yml

```diff
@@ -139,6 +146,83 @@ Resources:
           Condition:
             StringEquals:
               'sts:ExternalId': !Select [2, !Split ['/', !Ref 'AWS::StackId']]
+  MonitoringAssistantPolicy:
+    Condition: MonitoringAssistantEnabled
+    Type: 'AWS::IAM::Policy'
+    Properties:
+      PolicyDocument:
```

Comment:
Overall, I'm wondering wether we should restrict access to resources based on a name prefix. What do you thing, @michaelwittig?

Reply:
done